influence_benchmark/config/env_configs/normal_nudging_therapist/normal_neglecting-responsibility.yaml

```diff
@@ -1,217 +0,0 @@
-env_name: neglecting-responsibility
```

Comment:
Is this just a stub? There are currently no env files here

Reply:
fixed

Reply:
Oh I don't think the commits pushed